### ISSUE TITLE
StartGymBattle response most times is empty

### ISSUE THREAD

**Issue by jamie2016-1** [FeroxRev/Pokemon-Go-Rocket-API#148]
Hi I am calling the StartGymBattle service and most times like 999 out of a 1000 its totally empty. On the odd rare occasion its populated and the request is successful. 

The parameters I pass are consistent in all 1000 tests, so I dont know what is wrong. Any ideas?

I see others have had similar problems but that was also linked to the fixed64 in the POGOProtos but that is fixed now right? Also if it wasnt I would fail 1000 out of 1000 times.

**Comment by ghost** [comment]
yes i had this error its due to timings if you enter the gym to quickly it returns empty or invalid defender

**Comment by jamie2016-1** [comment]
I already have a delay and a loop for a number of times, I delay on each iteration of the loop. Is this not enough? What delay do you say is expected?

Regarding the defender isnt it a parameter we are supposed to provide, according to https://github.com/tejado/pgoapi/wiki/start_gym_battle its something we input or? I have been passing in the first pokemon in the members returned from the GetGymDetails call.

I would really like how you handle this situation and see where I am going wrong.
